src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import './App.css';
+import NavBar from './components/NavBar';
+import ListBooks from './components/ListBooks';
+import AddNewBook from './components/AddNewBook';
+
+const App = () => (
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<NavBar />}>
+        <Route
+          index
+          element={(
+            <>
+              <ListBooks />
+              <AddNewBook />
+            </>
+        )}
+        />
+        <Route path="categories" element={<div>Coming soon... categories</div>} />
```

Comment:
- [ ] good use of React Router :+1:

Reply:
@VICTOR-CHUKWU, Thank you very much for the feedback.